media-sound/deadbeef/files/deadbeef-1.9.5-clang-strict.patch

```diff
@@ -0,0 +1,14 @@
+https://bugs.gentoo.org/908414
+https://github.com/DeaDBeeF-Player/deadbeef/issues/2987
+
+--- a/configure.ac
++++ b/configure.ac
+@@ -16,7 +16,7 @@ AC_PROG_INSTALL
+ dnl AC_PROG_LIBTOOL
+ AC_CONFIG_MACRO_DIR([m4])
+ AC_C_BIGENDIAN
+-AM_GNU_GETTEXT
++AM_GNU_GETTEXT([external])
```

Comment:
This is not good either.  If you don't add an `AM_GNU_GETTEXT_REQUIRE_VERSION`, autopoint will not install the latest gettext is finds, and we will be left with 0.18.  It compiles fine, but it does not install any .mo files:
```
 *  FILES:-usr/share/locale/be/LC_MESSAGES/deadbeef.mo
 *  FILES:-usr/share/locale/bg/LC_MESSAGES/deadbeef.mo
...
 *  FILES:-usr/share/locale/zh_CN/LC_MESSAGES/deadbeef.mo
 *  FILES:-usr/share/locale/zh_TW/LC_MESSAGES/deadbeef.mo
 *   SIZE: 4.59MiB -> 2.94MiB, 84 -> 36 files
 * ------> FILES(-48) SIZE(-36.01%)
```

Reply:
Sorry for the delay, I was able to test this on my glibc system and there are no changes in the installed files with and without my current changes which include the `.mo` files. However on my musl system there are indeed no `.mo` files, but I am not sure if that is related to these changes? Using `AM_GNU_GETTEXT_REQUIRE_VERSION` made no difference on my musl box.

Reply:
`m4/gettext.m4` checks for `_nl_msg_cat_cntr`, which does not exist on musl.  Newer versions have a better check.  If that check fails, it behaves as if called with `--disable-nls`.
If `AM_GNU_GETTEXT_REQUIRE_VERSION` is used, `autopoint --force` should overwrite `gettext.m4` from `m4/` with a newer version and solve the issue.  I don't know why it doesn't on your machine.  Can you run `ebuild ... prepare` and check the revision of `m4/gettext.m4`?